pages/community/community.html

```diff
@@ -190,8 +193,13 @@ <h3><a href="https://forum.godotengine.org/">Godot Forum</a></h3>
 						loading="lazy">
 				</a>
 				<div class="base-padding">
+<<<<<<< Updated upstream
 					<h3><a href="https://discord.gg/godotengine">Discord</a></h3>
 					<p>A space to socialize with other community members.</p>
+=======
+					<h3><a href="https://discord.gg/bdcfAYM4W9">Discord</a></h3>
+					<p>Voice and text channels for socializing, announcements you can follow with your own servers, and more.</p>
+>>>>>>> Stashed changes
```

Comment:
Merge conflict not properly resolved.

Reply:
I honestly don't even know how I managed to create a merge conflict with simple text updates - I did pull from origin before changing things but oh well. At least this should be easy to fix:

```suggestion
<h3><a href="https://discord.gg/godotengine">Discord</a></h3>
					<p>Voice and text channels for socializing, announcements you can follow with your own servers, and more.</p>
>>>>>>> Stashed changes
```